src/lib.rs

```diff
@@ -1,70 +1,103 @@
-use image::{imageops::overlay, load_from_memory, ImageError, RgbaImage};
-use rand::random;
+use image::{imageops::overlay, ImageError, DynamicImage};
 use std::path::Path;
+use rand::prelude::*;
+use std::time::Instant;
 
 mod emoji;
 
-fn sums_chunked(samples_a: &[u8], samples_b: &[u8]) -> (i32, i32, i32) {
+
+fn measure_dist_chunks(samples_a: &[u8], samples_b: &[u8]) -> i64 {
     samples_a
-        .chunks_exact(3)
-        .zip(samples_b.chunks_exact(3))
-        .fold((0, 0, 0), |(r, g, b), (p_a, p_b)| {
-            (
-                r + (p_a[0] as i32 - p_b[0] as i32).abs(),
-                g + (p_a[1] as i32 - p_b[1] as i32).abs(),
-                b + (p_a[2] as i32 - p_b[2] as i32).abs(),
-            )
+        .chunks_exact(1)
+        .zip(samples_b.chunks_exact(1))
```

Comment:
```suggestion
        .zip(samples_b.iter())
```
i don't think you need to use `chunks_exact` if you are only taking one chunk